features/step_definitions/basic_steps.rb

```diff
@@ -90,6 +90,8 @@ def goto(page)
     user_registration_path
   when 'create restaurant'
     new_restaurant_path
+  when 'dishes'
+    dishes_path
```

Comment:
Try:

21  When I am on the menu page for "Lunch"
22  Then I should see "Starters"

instead of expecting to be on /dishes. I think /dishes is only used when creating a new dish, and if it works one should see the actual dish, not stay